legacy_plugins/whitelabelling_bar/README.md

```diff
@@ -0,0 +1,285 @@
+# 1. Plugin - Whitelabelling Bar
+
+This plugin adds a fixed bar to the bottom of the dashboard. The bar can display any text or an image or both
+depending on what is provided via plugin parameters.  
+The plugins is built to work with both Tiger/Panther and Bear backends.
+
+The main use case for this plugin is to add custom branding that is always visible on the dashboards even when
+embedded using iFrame.
+
```

Comment:
@dremileta please add here a section on how to work with the plugin as is in other open-sourced plugins, you can do it similarly as here: https://github.com/gooddata/gooddata-dashboard-plugins/tree/master/polar_area_chart_plugin#how-to-work-with-polar-area-chart-plugin or https://github.com/gooddata/gooddata-dashboard-plugins/tree/master/polar_area_chart_plugin#how-to-work-with-polar-area-chart-plugin.

We can take it as a completetion of the last point in the review ticket https://gooddata.atlassian.net/browse/SPACE-2568